ros_gz_sim/launch/ros_gz_sim.launch

```diff
@@ -1,26 +1,7 @@
 <launch>
-  <arg name="name" />
-  <arg name="config_file" default="" />
-  <arg name="container_name" default="ros_gz_container" />
-  <arg name="namespace" default="" />
-  <arg name="use_composition" default="True" />
-  <arg name="use_respawn" default="False" />
-  <arg name="log_level" default="info" />
-  <arg name="world_sdf_file" default="empty.sdf" />
-  <arg name="world_sdf_string" default="" />
-  <gz_server
-    world_sdf_file="$(var world_sdf_file)"
-    world_sdf_string="$(var world_sdf_string)"
-    container_name="$(var container_name)"
-    use_composition="$(var use_composition)">
-  </gz_server>
+  <gz_server 
+    world_sdf_file="empty.sdf" />
```

Comment:
Is it still possible to pass this launch file some of the arguments? i.e, can we call `ros2 launch ros_gz_sim.launch world_sdf_file=path/to/file.sdf`? I'm not, I don't see why we need this file at all.

Reply:
This change was unintentional. Fixed in https://github.com/gazebosim/ros_gz/pull/603.